testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2997,6 +2997,19 @@ public void testInsertArray()
         }
     }
 
+    @Test
+    public void testInsertSameValues()
+    {
+        skipTestUnless(hasBehavior(SUPPORTS_INSERT));
+
+        try (TestTable table = new TestTable(
+                getQueryRunner()::execute,
```

Comment:
since this uses `getQueryRunner::execute` doesn't it mean that `skipTestUnless` must use `SUPPORTS_CREATE_TABLE_WITH_DATA`?

Reply:
I followed ` @Test [..] testInsertArray()`